### DISCUSSION TITLE
Cinnamon panel weirdness

### DISCUSSION THREAD

**Discussion by Qronikarz** [linuxmint/discussions#411] [Questions]
I decided to challenge my habits and place the panel in different place than the so called "windows" like bottom. Top is definitely dumber and with left the menu button stays and maximized windows still have the close button in the top right so some habits stay the same. (and the space saved with the same panel height - 40x1080 vs 40x1920)
Anyway, let's get to the point.

This is the default on FHD screen (I used the right mouse click troubleshooting and reset to default option):
![default](https://github.com/linuxmint/discussions/assets/84787215/3f9bf718-953a-4f88-bca3-b48b347587b1)

This is the left side after I arranged everything so it has the same placement as the bottom, but vertically:
![leftpanel](https://github.com/linuxmint/discussions/assets/84787215/af377633-9036-42c7-882f-f7868396f779)
Notice the clock being broken into hours and minutes. You can fix it with that:
![calendar](https://github.com/linuxmint/discussions/assets/84787215/41b40f95-8c72-4116-8cc4-dab8130d9d35)
Now it's pretty much the same, but you can see the difference with the icons below the clock. Why are they so big? I only moved the icons, no settings were changed inside. What's even weirder is after I manually returned to old panel the icon size stayed the same which is why I used the reset to default option. So my question, can you somehow make them look exactly like in the default? Yes, there is the option to change their sizes, but some icons stay big:
![why](https://github.com/linuxmint/discussions/assets/84787215/a395391e-fb6b-4c83-b842-7d80f5a3334c)

And last issue. Is there any way to change the menu layout without actually making extension or plugin or anything like that.
![menu](https://github.com/linuxmint/discussions/assets/84787215/07931044-131f-4a54-8073-9b3b52cca838)
Personally I don't use the favorites that much and they take too much space, but if you turn them off the shutdown button also goes away. And I would want it to stay somewhere. Would it be possible to allow users to place the icons with a little more freedom? Here's something I would like to see:
![concept](https://github.com/linuxmint/discussions/assets/84787215/b04c75f0-cf60-4706-9f96-07e5818525af)
I'd also decrease the padding a little, but that's how I'd set up my menu. As you can see, the search bar could also be placed in different place.

**Comment by Jopp-gh** [comment]
On Mac, *menulets* (called applets on Mint) use usually  another bar than *applications*. 

1. First, application icons should be bigger - at least twice the size of applets. There is a core applet to regulate the common size of applets, furthermore you can resize elements in any panel (left, middle, right) by a specified *font size*. 
 
2. To quickly access applications, Mac places them at the bottom center (bar), applets instead, are placed on the top right (bar)

3. You can improve your configuration even further by moving very important applets in the middle of your top bar

After trying various configurations I understood  Apple's design choices, they aren't random.
